airbyte-cdk/java/airbyte-cdk/build.gradle

```diff
@@ -10,6 +10,10 @@ allprojects {
     version = getCdkTargetVersion()
 }
 
+java {
+    withSourcesJar()
```

Comment:
I'm still not convinced if this is working as expected because we change the artifact names before publish so I'll try it in a different PR. will remove it for now.